accepted/00053-disallow-downloading-when-patching.md

```diff
@@ -0,0 +1,61 @@
+- Feature Name: disallow-downloading-when-patching
+- Start Date: 29-03-2919
+- RFC PR: https://github.com/uyuni-project/uyuni-rfc/pull/6
+
+# Summary
+Optionally fail package installations and patch applications if they require any downloading.
+
+This is limited to minions on SUSE-based distros.
+
+# Motivation
+Minions with limited connectivity options might have little bandwidth available or bandwidth usage could come with a high cost (eg. in remote areas). In those cases we want users to be fully in control of when downloading happens, eg., to schedule that during off-peak hours (see https://github.com/uyuni-project/uyuni-rfc/pull/6). This RFC is about giving users the possibility to prevent package managers' default behavior of downloading at installation time.
+
```

Comment:
Don't you mean to refer to #5 instead of this one?

Reply:
Yes, thanks for spotting the typo. Corrected.